tests/yadbox/test_export.py

```diff
@@ -17,6 +17,8 @@ def test_pineappl_sf(tmp_path: pathlib.Path):
         "F2_light": [{"x": 0.1, "Q2": 10.0}],
         "F2_total": [{"x": 0.2, "Q2": 20.0}],
     }
+    # TODO: move this to banana
```

Comment:
let's remember that TODO

Reply:
Thanks for pointing it out! This is no longer needed: https://github.com/NNPDF/yadism/blob/bb150a1cf509711ddc209bc579cbca0cda65a08a/src/yadism/input/compatibility.py#L146